quickfixj-core/src/main/doc/usermanual/usage/configuration.html

```diff
@@ -456,6 +456,14 @@ <H3>QuickFIX Settings</H3>
     <TD> Y<br>N</TD>
     <TD> Y </TD>
   </TR>
+  <TR ALIGN="left" VALIGN="middle">
+      <TD> <I>ProcessMessageWithInvalidChecksum</I> </TD>
```

Comment:
Just a gut feeling, but I think this setting should rather be named `ValidateChecksum`. Unfortunately the logic and default value needs to be the other way round then.
The flags and methods inside the Message class are also named `validateChecksum` already.
Maybe wait some days if one of the other regulars has another opinion on this. @philipwhiuk maybe?

Reply:
Validate Checksum works for me as well. I will wait for opinion from others.

Reply:
I think you can go ahead.
Thanks